### DISCUSSION TITLE
Install disabled package protobuf@3.6

### DISCUSSION THREAD

**Discussion by BarnabasSzabolcs** [Homebrew/discussions#3583] [Casks] [answered]
Hi, how can I install protobuf@3.6?
Is there any tap that supports that?
My gcld3 python library is built against libprotobuf.30.dylib so that is the only protobuf I can use...

**Comment by p-linnane** [comment]
You can create your own tap and host anything you'd like. Details can be found [here](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).

**Comment by BarnabasSzabolcs** [comment, marked as answer]
I could install the formula by removing the deprecation line from the ruby file. :)))

But, it turns out, it has libprotobuf.17.dylib not libprotobuf.30.dylib so it does not work for me :(

**Reply by carlocab** [reply]
Note: be sure to undo the removal of the deprecation line when you're done, or this _will_ cause you `brew update` problems down the road.

**Reply by BarnabasSzabolcs** [reply]
I will, thanks for the heads up, Carlo!
